install_config/aggregate_logging.adoc

```diff
@@ -1011,6 +1012,121 @@ $ for dc in $(oc get deploymentconfig --selector logging-infra=elasticsearch -o
   done
 ----
 
+[[aggregated-logging-hostPath-Elasticsearch]]
+===== Configuring hostPath storage for Elasticsearch =====
+
+You can provision {product-title} clusters using hostPath storage for Elasticsearch.
+
+To use a local disk volume on each node host as storage for an Elasticsearch replica:
+
+. Create a local mount point on each infrastructure node for the local Elasticsearch storage:
++
+----
+mkdir /usr/local/es-storage
+----
+
+. Create a filesystem on the Elasticsearch volume:
++
+----
+mkfs.ext4 /dev/xxx
+----
+
+. Mount the elasticsearch volume:
++
+----
+mount /dev/xxx /usr/local/es-storage
+----
+
+. Add the following line to `/etc/fstab`:
++
+----
+/dev/xxx /usr/local/es-storage ext4
+----
+
+. Change ownership for the mount point:
++
+----
+chown 1000:1000 /usr/local/es-storage
+----
+
+. Give the privilege to mount and edit a local volume to the relevant service account:
++
+----
+  $ oc adm policy add-scc-to-user privileged  \
+         system:serviceaccount:logging:aggregated-logging-elasticsearch
+----
+  Use the project you created earlier (for example, logging) when running the logging playbook.
+
+. To claim that privilege, patch each Elasticsearch replica definition, as shown in the example,
+which specifies `--selector component=es-ops` for an Ops cluster:
++
+----
+  $ for dc in $(oc get deploymentconfig --selector component=es -o name);
+do
+    oc scale $dc --replicas=0
+    oc patch $dc \
+       -p '{"spec":{"template":{"spec":{"containers":[{"name":"elasticsearch","securityContext":{"privileged":
+true}}]}}}}'
+done
+----
++
```

Comment:
I'm not certain this is necessary, but it doesn't appear to break the ordering of the items.

Reply:
STET because there is only one "+" in the Asciidoc file, and as you said, it didn't seem to break order of things. So there remains just one "+" in the file at that line.